hr_contract_hourly_rate/models/hr_contract.py

```diff
@@ -26,12 +26,13 @@ class hr_contract(models.Model):
 
     salary_computation_method = fields.Selection(
         [('yearly', 'Annual Wage'),
-         ('monthly', 'Annual Wage'),
+         ('monthly', 'Monthly Wage'),
          ('hourly', 'Hourly Wage')],
         string='Salary Computation Method',
         help="Whether to use the annual wage or an hourly rate "
              "for computation of payslip.",
-        required=True, default='monthly')
+        required=True,
+        default='yearly')
```

Comment:
@tafaRU well think it changes from a country to another. In Canada, it is clearly yearly but I can not speak for the whole planet. Lets make a vote.